templates/repo/settings/protected_branch.tmpl

```diff
@@ -49,8 +49,7 @@
 								<div class="menu">
 									{{range .Users}}
 										<div class="item" data-value="{{.ID}}">
-											{{avatar $.Context . 28 "mini"}}
-											{{.GetDisplayName}}
+											{{avatar $.Context . 28 "mini"}}{{template "repo/search_name" Dict "Name" .Name "FullName" .FullName}}
```

Comment:
Why not use pass the `user` variable into it? 

`{{template "repo/search_name" .}}`